src/components/manage/Controlpanels/Controlpanel.jsx

```diff
@@ -141,6 +141,35 @@ class Controlpanel extends Component {
   }
   form = React.createRef();
 
+  // Goal would be to replace the Form schema prop in line 182, with this function
+  filterSchema = (controlpanel) => {
```

Comment:
@danalvrz I would move this function outside of the Controlpanel class, since it doesn't use other things from the class.